vrt-examples/src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Reminder that you still wanted to rename `vrt-examples` to something else.

I wonder whether we could eventually somehow turn this into a reusable package we publish on npm? I don't have too clear of a picture of what it would entail though.

Reply:
yeah, let's do some brainstorming on naming. Shopify have a folder called `playground` and a folder called `examples`. We might even just want to merge this with our current playground setup.